otter/http.py

```diff
@@ -113,7 +122,7 @@ def service_request(
         service_type, method, url, headers=None, data=None,
         log=None,
         reauth_codes=(401, 403),
-        success_codes=(200,),
+        success_pred=has_code(200),
```

Comment:
Ah, right; my bad. I saw that signature and forgot that `ServiceRequest` was already there, next to the old request_fn stuff :-)